core/services/mongodb-remote/build.gradle

```diff
@@ -12,6 +12,7 @@ buildscript {
 dependencies {
```

Comment:
Weird place for the comment but make sure to add the jnidispatch files back into the right places on android